### ISSUE TITLE
Firefox : "write for Wellcome collection" link on stories page is not visible to screen reader

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6003]
**What**
The first paragraph on the stories page has a link to a page outlining how to write for Wellcome collection. However the link is not visible to screen readers
<img width="1118" alt="Screenshot 2021-01-21 at 16 01 17" src="https://user-images.githubusercontent.com/7315126/105376690-ea444f00-5c01-11eb-9dcf-9cd02a74c90d.png">

**Where**

**Comment by jennpb** [comment]
@gestchild @davidpmccormick Can you check whether this is still a problem? We redesigned the page header since this was reported.

If not replicable, please go ahead and close it.

**Comment by gestchild** [comment]
I've tested this with VoiceOver and it works fine, the html also looks good so I can't see it causing issues for any other screen reader.
